### PR TITLE
OSDOCS#13000: Clarify platforms for CP and nodepools in IBM-Z/P

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.17+

Issue: https://issues.redhat.com/browse/OSDOCS-13000

Link to docs preview: https://86619--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-prepare/hcp-requirements.html#hcp-matrix-platform_hcp-requirements

https://86619--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hosted-control-planes-release-notes.html#hcp-tech-preview-features_hosted-control-planes-release-notes 

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: hosted_control_planes/hosted-control-planes-release-notes.adoc [hunk 1]
```diff
@@ -138,6 +138,11 @@ To resolve these issues, configure the DNS server settings for a dual stack netw
 
 Features which are Generally Available (GA) are fully supported and are suitable for production use. Technology Preview (TP) features are experimental features and are not intended for production use. For more information about TP features, see the link:https://access.redhat.com/support/offerings/techpreview[Technology Preview scope of support on the Red Hat Customer Portal].
 
+[IMPORTANT]
+====
+For {ibm-power-title} and {ibm-z-title}, you must run the control plane on machine types based on 64-bit x86 architecture and node pools on {ibm-power-title} or {ibm-z-title}.
```
Comment (snarayan-redhat) [inline hosted_control_planes/hosted-control-planes-release-notes.adoc:143]:
```suggestion
For {ibm-power-title} and {ibm-z-title}, you must run the control plane on machine types based on 64-bit x86 architecture, and node pools on {ibm-power-title} or {ibm-z-title}.
```

File: modules/hcp-support-matrix.adoc [hunk 2]
```diff
@@ -84,7 +84,14 @@ The following table maps {mce-short} versions to the hosted cluster versions tha
 [id="hcp-matrix-platform_{context}"]
 == Hosted cluster platform support
 
-The following table indicates which {product-title} versions are supported for each platform of {hcp}. In the table, _Management cluster version_ refers to the {product-title} version where the {mce-short} is enabled:
+The following table indicates which {product-title} versions are supported for each platform of {hcp}.
+
+[IMPORTANT]
+====
+For {ibm-power-title} and {ibm-z-title}, you must run the control plane on machine types based on 64-bit x86 architecture and node pools on {ibm-power-title} or {ibm-z-title}.
```
Comment (snarayan-redhat) [inline modules/hcp-support-matrix.adoc:91]:
```suggestion
For {ibm-power-title} and {ibm-z-title}, you must run the control plane on machine types based on 64-bit x86 architecture, and node pools on {ibm-power-title} or {ibm-z-title}.
```

File: modules/hcp-support-matrix.adoc [hunk 3]
```diff
@@ -84,7 +84,14 @@ The following table maps {mce-short} versions to the hosted cluster versions tha
...
+====
+
+In the table, _Management cluster version_ refers to the {product-title} version where the {mce-short} is enabled:
```
Comment (snarayan-redhat) [inline modules/hcp-support-matrix.adoc:94]: Per ISG, column names in a text do not need highlighting. 

> If the name is difficult to distinguish from the
surrounding text, enclose the name in double quotation
marks.

I think there we wouldn't need `"` either. But up to you.

```suggestion
In the following table, Management cluster version refers to the {product-title} version where the {mce-short} is enabled:
```

I understand this is an existing content that was moved. But just to ensure that this adheres guidelines, you could consider this comment.

**Review by snarayan-redhat — COMMENTED** [review]
Couple of nits. Otherwise LGTM!

**Comment by xenolinux** [comment]
Thanks for the review. Addressed suggestions.

**Pull request merged**